src/benchmarks/gc/GCPerfSim/GCPerfSim.cs

```diff
@@ -1854,8 +1861,8 @@ private static Args ParseFromCommandLine(string[] args)
             double a31 = 1;
             double a32 = 1;
             double a33 = 1;
-            double b1 = lohAllocRatioArg * overhead;
-            double b2 = pohAllocRatioArg * overhead;
+            double b1 = 1000 * lohAllocRatioArg * overhead;
```

Comment:
Since this is an esoteric calc, we should probably add a comment explaining why the 1000x is needed.

Reply:
The documentation for all of this code is in https://cshung.github.io/posts/poh-tuning-2/.  @cshung any chance you could update your blog post to include this fix?  (the two `+ O` under "Since we assumed the weights..." should be `+ 1000 O` and the two `R_{xoh}O` under "Now we think of..." should be `1000 R_{xoh}O`)

Reply:
> @cshung any chance you could update your blog post to include this fix? 

The blog is updated.